types/auth0-js/index.d.ts

```diff
@@ -536,7 +536,23 @@ export interface AuthOptions {
      */
     maxAge?: number;
     leeway?: number;
-    plugins?: any[];
+    jwksURI?: string;
+    overrides?: {
+      __tenant: string;
+      __token_issuer: string;
+    };
+    plugins?: any;
+    popupOrigin?: string;
+    protocol?: string;
+    response_type?: string;
+    rootUrl?: string;
+    state?: string;
+    tenant?: string;
+    token_issuer?: string;
```

Comment:
same here, can this one be `write`?:
https://github.com/auth0/auth0.js/blob/1524635f8c173847fa95edd823fddf85b6e4861b/src/web-auth/index.js#L157

Reply:
same here, maybe too restrictive with the usage. We coudl close this one too.